compose-files/docker-compose.api.yml

```diff
@@ -0,0 +1,46 @@
+version: "3"
+
+services:
+
+
+
+  ### kong api gateway
+  kong-database:
```

Comment:
why do you need kong here?

Reply:
is not used at the moment, we can also create an archive folder for somehting like this, because I am not sure if we needed anymore but if I wouldn't search everything again